cassandra-analytics-common/src/main/java/org/apache/cassandra/spark/utils/ByteBufferUtils.java

```diff
@@ -30,20 +30,13 @@
 import java.nio.charset.CharsetDecoder;
 import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
-
-import io.netty.util.concurrent.FastThreadLocal;
+import java.util.function.Supplier;
 
 public final class ByteBufferUtils
 {
+    public static final ThreadLocal<CharsetDecoder> UTF8_DECODER_PROVIDER = ThreadLocal.withInitial(StandardCharsets.UTF_8::newDecoder);
```

Comment:
This line seems similar to what is going on in `FastThreadLocalUtf8Decoder`, can it be referenced and reused here?

Reply:
Any reason we need to make it public? I don't see it being used anywhere else.